scout/server/blueprints/variants/controllers.py

```diff
@@ -1097,17 +1097,25 @@ def verification_email_body(case_name, url, display_name, category, subcategory,
     return html
 
 
-def cancer_variants(store, request_args, institute_id, case_name, form):
+def cancer_variants(store, request_args, institute_id, case_name, form, page=1):
     """Fetch data related to cancer variants for a case."""
+
     institute_obj, case_obj = institute_and_case(store, institute_id, case_name)
-    variants_query = store.variants(case_obj['_id'], category='cancer', query=form.data).limit(50)
+    per_page = 50
+    skip_count = per_page * max(page - 1, 0)
+    variants_query = store.variants(case_obj['_id'], category='cancer', query=form.data)
+    variant_count = variants_query.count()
```

Comment:
@moonso won't be happy about that one; ´´´.count()´´´ is apparently going deprecated in pymongo. But, we have the same for the other variants-controllers so we may as well change them all at once! Do keep it.

Reply:
I see. What do we use instead?

Reply:
Okej. Good to know